libexec/tachyon-config.sh

```diff
@@ -27,14 +27,17 @@ if [ -z "$TACHYON_SYSTEM_INSTALLATION" ]; then
   export TACHYON_CONF_DIR="$TACHYON_HOME/conf"
   export TACHYON_LOGS_DIR="$TACHYON_HOME/logs"
   export TACHYON_JARS="$TACHYON_HOME/assembly/target/tachyon-assemblies-${VERSION}-jar-with-dependencies.jar"
-  export JAVA="$JAVA_HOME/bin/java"
 fi
 
-# Environment settings should override * and are administrator controlled.
-if [ -e $TACHYON_CONF_DIR/tachyon-env.sh ] ; then
-  . $TACHYON_CONF_DIR/tachyon-env.sh
+# Make sure tachyon-env.sh exists  
+if [ ! -e $TACHYON_CONF_DIR/tachyon-env.sh ]; then
+  echo "Cannot find tachyon-env.sh in $TACHYON_CONF_DIR."
+  echo "Please create one manually or using $TACHYON_HOME/bin/tachyon bootstrap-conf function."
+  exit 0
```

Comment:
`using command: $TACHYON_HOME/bin/tachyon bootstrap-conf`, the suffix `function` would confuse user to think it is part of the command.  Also, `exit 1` since the script fails.